src/Functions/stringBytesUniq.cpp

```diff
@@ -0,0 +1,52 @@
+#include <Columns/ColumnConst.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnVector.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/stringBytes.h>
+#include <Functions/IFunction.h>
+#include <Common/BitHelpers.h>
+#include <Common/PODArray.h>
+
+#include <bit>
+#include <cmath>
+
+namespace DB
+{
+
+struct StringBytesUniqImpl
+{
+    using ResultType = UInt16;
+
+    static ResultType process(const UInt8 * data, size_t size)
+    {
+        UInt64 mask[4] = {0};
+        const UInt8 * end = data + size;
+
+        for (; data < end; ++data)
+        {
+            UInt8 byte = *data;
+            mask[byte >> 6] |= (1ULL << (byte & 0x3F));
+        }
+
+        return std::popcount(mask[0]) + std::popcount(mask[1]) + std::popcount(mask[2]) + std::popcount(mask[3]);
+    }
+};
+
+
+struct NameStringBytesUniq
+{
+    static constexpr auto name = "stringBytesUniq";
+};
+
+
+using FunctionStringBytesUniq = FunctionStringBytes<StringBytesUniqImpl, NameStringBytesUniq>;
+
+REGISTER_FUNCTION(StringBytesUniq)
+{
+    factory.registerFunction<FunctionStringBytesUniq>(
+        FunctionDocumentation{.description = R"(Counts the number of distinct bytes in a string.)"});
```

Comment:
Please provide the complete in code documentation as described the README for both the functions: https://github.com/ClickHouse/ClickHouse/blob/e916901443b1b1eeddbdcf9de48adc9cd62cf6a5/src/Functions/README.md. Currently the docs check are failing because of that. You can look at other similar functions for example.
```
/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/src/Functions/stringBytesUniq.cpp:49:99: error: missing field 'category' initializer [-Werror,-Wmissing-designated-field-initializers]
           49 |         FunctionDocumentation{.description = R"(Counts the number of distinct bytes in a string.)"});
              |                                                                                                   ^
        1 error generated.
```

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/79350/commits/7e26ad41c29cc02d6185cd7d523b4f111ab862ed